docs/siem/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -0,0 +1,387 @@
+[[exceptions-api-create-exception-item]]
+=== Create exception item
+
+Creates an exception item and associates it with the specified
+<<exceptions-api-create-container, exception container>>.
+
+See <<lists-api-overview>> for information about creating exception items from
+lists, such as a list of IP addresses or host names.
+
+NOTE: Before creating exception items, you must create an exception container.
+
+[IMPORTANT]
+==============
+Endpoint rule exception items cannot use
+<<lists-api-overview, lists>> (the `list` in the `entries` array), and the
+following fields cannot be used in exception queries (as `field` values in the
+`entries` object):
+
+* `file.Ext.quarantine_path`
+* `file.Ext.quarantine_result`
+* `process.entity_id`
+* `process.parent.entity_id`
+* `process.ancestry`
+==============
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists/items`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
+
+|No, defaults to empty array.
+
+|`description` |String |Describes the exception item. |Yes
+|`entries` |<<entries-object-schema, entries[]>> |Array containing the
+exception queries. Boolean `AND` logic is used to evaluate the relationship
+between array elements. If you want to use `OR` logic, create a separate
+exception item. |Yes
+|`list_id` |String |ID of the associated <<exceptions-api-create-container, exception container>>. |Yes
+|`item_id` |String |Unique identifier of the exception item. |No, automatically
+created when it is not provided.
+|`meta` |Object |Placeholder for metadata about the exception item. |No
+|`name` |String |The exception item's name. |Yes.
+|`namespace_type` |String a|Determines whether the exception item is available
```

Comment:
Wondering if we should specify that it should match the parent? @FrankHassanabad is that the case that the items namespace_type should match the parent's?